resources/popup.js

```diff
@@ -1,27 +1,30 @@
+// Use the toggle button on the popup to switch the loggingEnabled option.
+function toggle() {
+    chrome.storage.sync.get({ loggingEnabled: true }, toggleSetter);
+}
+
+// Helper function that sets loggingEnabled to the right new value.
+function toggleSetter(settingsObject) {
+    chrome.storage.sync.set({
+        loggingEnabled: !(settingsObject.loggingEnabled)
+    })
+}
+
 $(document).ready(function () {
```

Comment:
It runs from lines 13 till 30, so minus header and trailer that's only 16 lines.